ports/rp2/main.c

```diff
@@ -223,6 +227,12 @@ int main(int argc, char **argv) {
         soft_timer_deinit();
         gc_sweep_all();
         mp_deinit();
+        // Reset the MCU frequency
+        set_sys_clock_khz(125000, false);
```

Comment:
The intention here is to have the device at the same state as when entering the `for (;;) {` loop. If code e.g. in boot.py and main.py changes the frequency, it will still do it.